azblob/url_blob.go

```diff
@@ -73,7 +73,7 @@ func (b BlobURL) Download(ctx context.Context, offset int64, count int64, ac Blo
 	dr, err := b.blobClient.Download(ctx, nil, nil, nil,
 		httpRange{offset: offset, count: count}.pointers(),
 		ac.LeaseAccessConditions.pointers(), xRangeGetContentMD5, nil,
-		nil, nil, EncryptionAlgorithmNone, // CPK
+		cpk.EncryptionKey, cpk.EncryptionKeySha256, cpk.EncryptionAlgorithm, // CPK
```

Comment:
>pk.EncryptionKey, cpk.EncryptionKeySha256, cpk.EncryptionAlgorithm, // CPK [](start = 3, length = 74)

The encryption scope is missing

Reply:
Yeah. It's not supported in Blob Download. Check [this](https://github.com/Azure/azure-storage-blob-go/blob/dev/swagger/blob.json#L2900) 

And here's the generated function signature for blob download:

`func (client blobClient) Download(ctx context.Context, snapshot *string, versionID *string, timeout *int32, rangeParameter *string, leaseID *string, rangeGetContentMD5 *bool, rangeGetContentCRC64 *bool, encryptionKey *string, encryptionKeySha256 *string, encryptionAlgorithm EncryptionAlgorithmType, ifModifiedSince *time.Time, ifUnmodifiedSince *time.Time, ifMatch *ETag, ifNoneMatch *ETag, ifTags *string, requestID *string) (*downloadResponse, error)`